internal/service/project/data_source_project_test.go

```diff
@@ -36,9 +37,10 @@ func TestAccProjectDSProject_byID(t *testing.T) {
 					},
 				)),
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttrSet("mongodbatlas_project.test", "name"),
-					resource.TestCheckResourceAttrSet("mongodbatlas_project.test", "org_id"),
-					resource.TestCheckResourceAttr("mongodbatlas_project.test", "teams.#", "2"),
+					resource.TestCheckResourceAttrSet(dataSourceName, "name"),
+					resource.TestCheckResourceAttrSet(dataSourceName, "org_id"),
+					resource.TestCheckResourceAttr(dataSourceName, "teams.#", "2"),
+					resource.TestCheckResourceAttrSet(dataSourceName, "ip_addresses.services.clusters.#"),
```

Comment:
is it useful to check specific values?

Reply:
for acceptance tests I would need to create a cluster if I want to check more specific values, I prefer to rely on unit testing to ensure the model is correctly handling the SDK responses when ip addresses are returned.